zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/MappingCreatedHandler.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.handlers;
+
+import io.camunda.exporter.exceptions.PersistenceException;
+import io.camunda.exporter.store.BatchRequest;
+import io.camunda.webapps.schema.entities.usermanagement.MappingEntity;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.protocol.record.value.MappingRecordValue;
+import java.util.List;
+
+public class MappingCreatedHandler implements ExportHandler<MappingEntity, MappingRecordValue> {
+  private final String indexName;
+
+  public MappingCreatedHandler(final String indexName) {
+    this.indexName = indexName;
+  }
+
+  @Override
+  public ValueType getHandledValueType() {
+    return ValueType.MAPPING;
+  }
+
+  @Override
+  public Class<MappingEntity> getEntityType() {
+    return MappingEntity.class;
+  }
+
+  @Override
+  public boolean handlesRecord(final Record<MappingRecordValue> record) {
+    return getHandledValueType().equals(record.getValueType());
```

Comment:
🔧 As this handler is targeted to handle create we should also include the intent type in this check for example the user handler checks for Updated and Created ([REF](https://github.com/camunda/camunda/blob/main/zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/UserCreatedUpdatedHandler.java#L40-L44))

Reply:
I don't believe we can, when we handle the adding of the record to the batch we don't know what the intent is at that point (we are dealing with an entity), so at that point we wouldn't know whether to add/delete (at least that was true the last time I worked with the code)

Reply:
If it works for `UserCreatedUpdatedHandler` can work for `MappingCreatedDeletedHandler` right ?

Reply:
Created + updated can use the same `add` batch method method in the `flush` method, deleted needs to use the `delete` batch method on a batch request so the record is removed from the indexes.

Reply:
You're right, I missed it 💪